### DISCUSSION TITLE
Ability to add a Label to a page

### DISCUSSION THREAD

**Discussion by marcosgrimm** [GitbookIO/community#495] [Feature Requests]
I would like to suggest a feature where a Page can have one or more labels. This way it would be easier to categorize Pages that are related but not under the same Parent Page.

**Comment by addisonschultz** [comment]
Definitely a good idea. We've been investigating ways to categorize pages better, I'll make sure we add this feedback in!

**Comment by b9Joker108** [comment]
I agree, this would be very useful, and from this functionality, it should also be possible to generate a conceptual map.
